doc/sphinx/language/core/assumptions.rst

```diff
@@ -155,6 +155,12 @@ has type :n:`@type`.
    is closed, the :n:`@ident`\(s) become undefined and every object depending on them will be explicitly
    parameterized (i.e., the variables are *discharged*).  See Section :ref:`section-mechanism`.
 
+   :n:`:>`
+     If specified, the name of this field is automatically
+     declared as a coercion to the class of this
+     field type. Note that fields always verify the uniform
+     inheritance condition.
```

Comment:
Actually `field` is still language that is specific to records. I would put something like "the name before the `:>` is declared as a coercion to the class of its type". I would remove the note about the uniform inheritance condition from here and only link to the coercion chapter instead.